api/lib/domain/services/certification-candidates-ods-service.js

```diff
@@ -211,6 +213,21 @@ function _handleParsingError() {
   throw new CertificationCandidatesError({ code: 'INVALID_DOCUMENT', message: 'Le document est invalide.' });
 }
 
+function _handleDuplicateCandidate() {
+  const hashes = new Set();
+  return ({ candidate }) => {
+    const candidateInformationFootprint =
+      candidate.firstName.toLowerCase() + candidate.lastName.toLowerCase() + candidate.birthdate;
```

Comment:
Si on passe directement le candidat, on peut destructurer ici:
```suggestion
  return ({ firstName, lastName, birthdate }) => {
    const candidateInformationFootprint =
     firstName.toLowerCase() +lastName.toLowerCase() + birthdate;
```